src/components/LayoutTree.tsx

```diff
@@ -567,12 +573,16 @@ const LayoutTree = () => {
         ycol1Ancestors.push({ y: yaccum + height, depth: node.depth })
       }
 
+      const nextAutofocus = next?.autofocus
+      const nextIsLastVisible = nextAutofocus === 'show' || nextAutofocus === 'dim'
```

Comment:
This variable is called `nextIsLastVisible`, but it actually just checks `nextIsVisible`. There is no logic here related to it being the last thought or last visible thought.